src/components/Character/Character.jsx

```diff
@@ -55,28 +61,35 @@ export default function Character(props){
 			aria-roledescription="slide"
 			aria-label={`Character ${index + 1} of ${characterData.length}.`}
 			aria-hidden={ hidden }>
-			<article>
-				<h1 id={`character__${id}`}>
-					{ name }
-				</h1>
-				<h2>
-					{ nickname }
-				</h2>
-				<h3>
-					{ actor }
-				</h3>
-				<p>
-					{ body }
-				</p>
+			<article className={ s.container }>
+				<div className={ s.content }>
+					<div className={ s.text }> 
+						<h1 className={ s.name } 
+							id={`character__${id}`}
+						>
+							{ name }
+						</h1>
+						<h2 className={ s.nickName } >
+							{ nickname }
+						</h2>
+						<h3 className={ s.actorName } >
+							{ actor }
+						</h3>
+						<p className={ s.bio } > 
```

Comment:
when the renderBody updates are merged, don't forget to convert this to a `<div>` and render the `<p>`s as children instead it

Reply:
also, probs call that `s.body` innit.

Reply:
will do and done :)